src/main/java/com/github/mc1arke/sonarqube/plugin/CommunityBranchPlugin.java

```diff
@@ -67,8 +71,80 @@ public void load(CoreExtension.Context context) {
                 //the name and description shown on the UI are automatically loaded from core.properties so don't need to be specified here
                 PropertyDefinition.builder(CoreProperties.LONG_LIVED_BRANCHES_REGEX).onQualifiers(Qualifiers.PROJECT)
                         .category(CoreProperties.CATEGORY_GENERAL).subCategory(CoreProperties.SUBCATEGORY_BRANCHES)
-                        .defaultValue(CommunityBranchConfigurationLoader.DEFAULT_BRANCH_REGEX).build());
+                        .defaultValue(CommunityBranchConfigurationLoader.DEFAULT_BRANCH_REGEX).build(),
+
+                PropertyDefinition.builder("sonar.pullrequest.provider").category(PULL_REQUEST_CATEGORY_LABEL)
+                        .subCategory(GENERAL)
+                        .onlyOnQualifiers(Qualifiers.PROJECT).name("Provider").type(PropertyType.SINGLE_SELECT_LIST)
+                        .options("GitlabServer").build(),
+
+                PropertyDefinition.builder("sonar.pullrequest.base.image.url")
+                        .category(PULL_REQUEST_CATEGORY_LABEL)
+                        .onlyOnQualifiers(Qualifiers.PROJECT)
+                        .name("Base image URL")
+                        .defaultValue("https://raw.githubusercontent.com/mc1arke/sonarqube-community-branch-plugin/pr-decoration/src/main/resources/pr-decoration-images")
+                        .description("Default: https://raw.githubusercontent.com/mc1arke/sonarqube-community-branch-plugin/master/src/main/resources/pr-decoration-images")
+                        .type(PropertyType.STRING)
+                        .build(),
+
+                PropertyDefinition.builder("sonar.pullrequest.summary.comment.enabled")
+                        .category(PULL_REQUEST_CATEGORY_LABEL)
+                        .subCategory(GENERAL)
+                        .onlyOnQualifiers(Qualifiers.PROJECT)
+                        .name("Enable summary comment")
+                        .description("This enables the summary comment (if implemented).")
+                        .type(PropertyType.BOOLEAN)
+                        .defaultValue("true")
+                        .build(),
+
+                PropertyDefinition.builder("sonar.pullrequest.file.comment.enabled")
+                        .category(PULL_REQUEST_CATEGORY_LABEL)
+                        .subCategory(GENERAL)
+                        .onlyOnQualifiers(Qualifiers.PROJECT)
```

Comment:
Is this not something that should be global across all SonarQube projects, rather than specific to each project? If so, this should probably be:
```suggestion
                        .onlyOnQualifiers(Qualifiers.APP)
```

Reply:
using the same parameter as the bitbucket integration